pkg/controller/hyperconverged/factory.go

```diff
@@ -73,6 +78,19 @@ func newCDIForCR(cr *hcov1alpha1.HyperConverged, namespace string) *cdi.CDI {
 	}
 }
 
+func newMachineRemediationOperatorCR(cr *hcov1alpha1.HyperConverged, namespace string) *mrv1.MachineRemediationOperator {
```

Comment:
You need to add your crd template to `templates/crds/...`.

Reply:
@rthallisey @davidvossel I thought we will take these templates from the CSV generator?

Reply:
I think you'll still need a machineRemediationCRD.yaml.in file to scoop up the content from the CSV generator.

Reply:
csv-generator generates also the CRD for the operator

Reply:
Right, but I'm referring to is https://github.com/kubevirt/hyperconverged-cluster-operator/blob/master/templates/crds/hco.crd.yaml.in.